internal/service/appautoscaling/scheduled_action.go

```diff
@@ -120,6 +120,15 @@ func resourceScheduledActionPut(ctx context.Context, d *schema.ResourceData, met
 		ScalableDimension:   aws.String(d.Get("scalable_dimension").(string)),
 	}
 
+	currentStartTime := getTimeIfExist(d, "start_time")
+	if currentStartTime != nil {
+		input.StartTime = aws.Time(currentStartTime.(time.Time))
+	}
+	currentEndTime := getTimeIfExist(d, "end_time")
+	if currentEndTime != nil {
+		input.EndTime = aws.Time(currentEndTime.(time.Time))
+	}
```

Comment:
To avoid unexpected missing StartTime and EndTime whenever we try to update existing ScheduledAction I tried to set existing value if exist, but if it doesn't exist then it will behave like current behavior, but if previously there is a value, then it will carry it.